pom.xml

```diff
@@ -40,7 +40,8 @@
 	<properties>
 		<maven.compiler.source>1.8</maven.compiler.source>
 		<maven.compiler.target>1.8</maven.compiler.target>
-		<spring-cloud-release.version>Finchley.M5</spring-cloud-release.version>
+        <spring-cloud-release.version>Finchley.M5</spring-cloud-release.version>
```

Comment:
Tabs?

Reply:
Fixed in https://github.com/spring-cloud/spring-cloud-gcp/pull/369/commits/5963a44ccee18d120866bdc7df39769f7448f455

Reply:
Late. See  what we have merged into the `master`: https://github.com/spring-cloud/spring-cloud-gcp/blob/master/pom.xml#L42 😄

Reply:
Sorry, saw outdated PR somehow.